src/plugins/console/public/application/components/console_menu.tsx

```diff
@@ -31,6 +31,7 @@ interface Props {
 interface State {
   isPopoverOpen: boolean;
   curlCode: string;
+  curlError: string | null;
```

Comment:
shouldnt this be of type `Error` ? 🤔

Reply:
Indeed, thanks for pointing it out! Updated it